src/consensus/params.h

```diff
@@ -31,6 +31,9 @@ struct BIP9Deployment {
     int64_t nStartTime;
     /** Timeout/expiry MedianTime for the deployment attempt. */
     int64_t nTimeout;
+
+    /** Constant for nTimeout very far in the future. */
+    static constexpr int64_t NO_TIMEOUT = 999999999999ULL;
```

Comment:
Micro nit: Still confusing value (int64 = unsigned long long). Not worth fixing unless other changes are needed too.

Reply:
Done, switched to numeric_limits.